docs/ui/components/PageHeader/PageTitleButtons.tsx

```diff
@@ -32,58 +33,70 @@ export function PageTitleButtons({ packageName, sourceCodeUrl }: Props) {
           </div>
         </Button>
       )}
-      {sourceCodeUrl && (
-        <Tooltip.Root delayDuration={500}>
+      {[
+        ...(sourceCodeUrl
+          ? [
+              {
+                icon: <GithubIcon className="mt-0.5 text-icon-secondary" />,
+                label: 'GitHub',
+                tooltip: (
+                  <>
+                    View source code of <MONOSPACE>{packageName}</MONOSPACE> on GitHub
+                  </>
+                ),
+                href: sourceCodeUrl,
+              },
+            ]
+          : []),
+        ...(sourceCodeUrl?.startsWith('https://github.com/expo/expo')
+          ? [
+              {
+                icon: <Tag03Icon className="mt-0.5 text-icon-secondary" />,
+                label: 'Changelog',
+                tooltip: (
+                  <>
+                    View the changelog of <MONOSPACE>{packageName}</MONOSPACE> on GitHub
+                  </>
+                ),
+                href: `${sourceCodeUrl}/CHANGELOG.md`,
+              },
+            ]
+          : []),
+        ...(packageName
+          ? [
+              {
+                icon: <BuildIcon className="mt-0.5 text-icon-secondary" />,
+                label: 'npm',
+                tooltip: 'View package in npm registry',
+                href: `https://www.npmjs.com/package/${packageName}`,
+              },
+            ]
+          : []),
+      ].map(({ label, icon, tooltip, href }) => (
+        <Tooltip.Root key={label} delayDuration={500}>
           <Tooltip.Trigger asChild>
             <Button
               theme="quaternary"
               className="min-h-[48px] min-w-[60px] justify-center px-2 max-xl-gutters:min-h-[unset]"
               openInNewTab
-              href={sourceCodeUrl}>
+              href={href}>
               <div
                 className={mergeClasses(
                   'flex flex-col items-center',
                   'max-xl-gutters:flex-row max-xl-gutters:gap-1.5'
                 )}>
-                <GithubIcon className="mt-0.5 text-icon-secondary" />
+                {icon}
                 <FOOTNOTE crawlable={false} theme="secondary">
-                  GitHub
+                  {label}
                 </FOOTNOTE>
               </div>
             </Button>
           </Tooltip.Trigger>
           <Tooltip.Content className="max-w-[300px]">
```

Comment:
Let's alter the TooltipContent (max)-width or changelog label value, so we avoid splitting into two lines:

<img width="740" height="300" alt="Image" src="https://github.com/user-attachments/assets/9c92a576-8e8c-45e1-a82c-ce71e2add2cc" />